File: trunk/Projects/Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/ItemManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface IItemManager : IManagerBase<Item, string>
    {
	}
	
	partial class ItemManager : ManagerBase<Item, string>, IItemManager
    {
	}
}

File: Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/ItemManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface IItemManager
	'	Inherits IManagerBase(Of Item, System.String)
	'End Interface

	Partial Class ItemManager
		Inherits ManagerBase(Of Item, System.String)
		Implements IItemManager
	End Class
End Namespace
